tests/baselines/reference/user/mqtt.log

```diff
@@ -0,0 +1,8 @@
+Exit Code: 2
+Standard output:
+node_modules/mqtt/types/lib/client-options.d.ts(3,31): error TS7016: Could not find a declaration file for module 'ws'. 'E:/Github/TypeScript/tests/cases/user/mqtt/node_modules/ws/index.js' implicitly has an 'any' type.
```

Comment:
`mqtt` specifies `@types/ws` as a `devDependency`, but not a `dependency`. https://github.com/mqttjs/MQTT.js/issues/1408 Looks like their current guidance is to tell users to install `@types/ws` themselves alongside `@types/node`, so our test needs to do the same.